module/framework.js

```diff
@@ -576,44 +578,46 @@ async function _getCategories({cache=true}={}) {
     id: CATEGORY_ID.uncategorized,
     title: game.i18n.localize('WA.CategoryUncategorized'),
     position: 9e9,
-    copyForSort: [],
-    children : [],
-    parent: root,
+    articles: [],
+    articleIds: [],
+    children: [],
+    childrenIds: [],
     isUncategorized: true
   };
   categories.set(uncategorized.id, uncategorized);
 
   // Retrieve categories from the World Anvil API (build map)
   const request = await anvil.getCategories();
-  for ( let c of (request?.categories || []) ) {
+  
+  // First loop : add id listings and store in map
+  const reqCategories = (request?.categories || []);
+  for ( let c of reqCategories ) {
```

Comment:
I think ideally the first loop would *only* handle `categories.set(c.id, c)` and leave the rest of the work for the second loop. I think this would be more clear.

Reply:
Yes, make sense.
It was doing something else before. But now, it doesn't make sense anymore.